### PR TITLE
task 3 & 4 completed

### PR DESCRIPTION
**Pull request by StevenJune**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+app.get("/api/topics", getTopics);
+app.get("/api/articles/:article_id",getArticlesById)
+
+// below lines will catch all the unmatch endpoints
+app.all("/*", (req, res) => {
+  res.status(404).send({ msg: "Route not found" });
+});
+
+app.use((err, req, res, next) => {
+  res.sendStatus(500);
```
Comment (Rayhaan2k9) [inline app.js]: We need to change this to `res.status(500).send("Internal server error")`

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            topic: 'mitch',
+            created_at: '2020-10-16T05:03:00.000Z',
+            votes:0,
+          });
+        });
+    });
```
Comment (Rayhaan2k9) [inline __tests__/app.test.js]: Great happy path tests, but we also need some error handling. What if endpoint is /api/articles/1000 or /api/articles/banana?

**Review by Rayhaan2k9 — CHANGES_REQUESTED** [review]
A couple things to fix Steven. I've left comments for you. Once fixed, please send another pull request. Also please try to work on each ticket in its own branch 🙂

**Review by leekli — APPROVED** [review]
Good work Steven, thank you for making the recommended changes.

You are now able to merge this.

**Pull request merged**